### DISCUSSION TITLE
transcoding?

### DISCUSSION THREAD

**Discussion by syadnom** [tgorgdotcom/locast2plex#134] [Q&A]
Anyone figured out how to transcode with ffmpeg instead of outputting mpegts?  plex' hdhomerun interface supports h.264 input.

**Comment by grantbarrett** [comment]
What about running a post-processing script? I use this one: https://github.com/nebhead/PlexPostProc

**Comment by seantdavidson** [comment]
@grantbarrett This is probably not the place to ask but do you have that post processing script running in a docker container? I have my Plex running in Docker and want to try out this script also having it in a Docker container
